scripts/tests.sh

```diff
@@ -73,4 +73,4 @@ do
 done
 
 # Run all tests all
-cargo test -workspace -F runtime-benchmarks,try-runtime
+cargo test --workspace -F runtime-benchmarks,try-runtime
```

Comment:
Was failing locally

Reply:
cc @lemunozm

Reply:
I should have added a typo after testing by myself it worked 😆